composed_api.rst

```diff
@@ -287,20 +276,20 @@ Endpoint
 
 ::
 
-    GET <instance_url>/<instance_root>_ba/api/v1/aoi/add
+    GET <instance_url>/<instance_root>_ba/api/v2/aoi/add
 
 Request Parameters
 ^^^^^^^^^^^^^^^^^^
 
 +-------------------+-------------------------------------------------------+
 | Query parameter   | Value                                                 |
 +===================+=======================================================+
+| source            | *Required*. Your GRiD generated API key.              |
```

Comment:
I think it was named source because our original intention was to be able to track the source of the API call.  I like the term "key" better though and wouldn't mind making the switch.  And we have made some changes to generating the keys for this version, it now gives you a more randomly generated key instead of being user specified (legacy user generated keys are still valid, however).  

That said, the goal of the API keys is to have them be at the application level.  We already have control over the user due to the login requirements, but we really wanted to see what applications were being used to connect to GRiD and to also be able to shut them off if they are behaving improperly.  It sounds like this should be made more clear?  Maybe we should include some more details in the API documentation itself?